Buildings/Fluid/AirFilters/BaseClasses/MassAccumulation.mo

```diff
@@ -1,13 +1,17 @@
 within Buildings.Fluid.AirFilters.BaseClasses;
 model MassAccumulation
   "Component that mimics the accumulation of the contaminants"
-  parameter Real mCon_nominal
-    "Maximum mass of the contaminant captured by the filter";
+  parameter Integer nin(
+    min=1)=1
+    "Number of input connections";
```

Comment:
I think you mean the total types of contaminant substances? So can we change the `nin` to `nConSub` and change the comment to `Total number of contaminant substance types`?

Reply:
Change it as suggested